project/model/src/main/java/vasilizas/bean/DefaultPerson.java

```diff
@@ -1,4 +1,4 @@
-package bean;
+package vasilizas.bean;
 
 public class DefaultPerson extends Person{
```

Comment:
Пробел перед скобкой
```suggestion
public class DefaultPerson extends Person {
```

Reply:
done